pkg/security/podsecuritypolicy/factory.go

```diff
@@ -51,11 +48,9 @@ func (f *simpleStrategyFactory) CreateStrategies(psp *policy.PodSecurityPolicy,
 	}
 
 	var groupStrat group.GroupStrategy
-	if utilfeature.DefaultFeatureGate.Enabled(features.RunAsGroup) {
```

Comment:
Shouldn't we keep this until the feature flag is removed in 1.22?

Reply:
This is example shows that it is not needed: https://github.com/kubernetes/kubernetes/pull/94140/ Flag stays to make sure kubelet starts with this flag set

Reply:
So no action required from me right ?